support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React, { type Node } from 'react';
+
+type LandingPage = 'digitalSubscription' | 'newspaper' | 'guardianWeekly';
+
+type TimeBoundCopy = {|
+  start: string;
+  end?: string;
+  copy: Node;
+|}
+
+type TimedCopyCollection = { [LandingPage]: TimeBoundCopy[] }
+
+const timedCopy: TimedCopyCollection = {
+  digitalSubscription: [
```

Comment:
nice! Just wondering, would it make sense to have a test that fails the build if there is a timed copy that has ended? a bit like the dotcom feature switches - to make sure things are cleaned up.

Reply:
I think for the moment we can probably do without- I don't think it's a huge issue if we have expired copy hanging around for a bit, and we'd likely just delete anything out of date once we added something new- but it could be useful to add something like that in the future!